vit/config_parser.py

```diff
@@ -350,8 +350,12 @@ def has_project_column(self, report_name):
         return self.get_column_index(report_name, 'project') is not None
 
     def has_primary_project_ascending_sort(self, report):
-        primary_sort = report['sort'][0]
-        return primary_sort[0] == 'project' and primary_sort[1] == 'ascending'
+        try:
+            primary_sort = report['sort'][0]
+        except KeyError:
+            return False
+        else:
```

Comment:
This ```else``` seems superfluous, given the ```return ``` above?

Reply:
Yes, you're right. I'll remove it.